tendrl/gluster_integration/objects/gluster_brick_path/__init__.py

```diff
@@ -0,0 +1,23 @@
+from tendrl.commons import objects
+
+
+class GlusterBrickDir(objects.BaseObject):
```

Comment:
My idea was to use the same object to store additional attributes related to gluster brick dir. you were hinting at having a list of user defined brick paths in future, which would also come under this object as a new attribute like: "user_brick_paths"
Please provide your thoughts.